app/src/main/java/br/org/cesar/knot_setup_app/fragment/login/LoginContract.java

```diff
@@ -0,0 +1,15 @@
+package br.org.cesar.knot_setup_app.fragment.login;
+
+public interface LoginContract {
+
+    interface ViewModel {
+        void fillEmail(String email);
+        void callbakcOninvalidCredentials();
```

Comment:
The `on` prefix already denotes a callback. No need to call it `callbackOnSomething`.

Reply:
Done.